utils/setup.cfg

```diff
@@ -7,10 +7,12 @@ universal=1
 [nosetests]
 tests=../,../roles/openshift_master_facts/test/,test/
 verbosity=2
-with_coverage=1
-cover_html=1
-cover_package=ooinstall
-cover_min_percentage=70
+with-coverage=1
+cover-html=1
+cover-inclusive=1
+cover-min-percentage=70
+cover-erase=1
```

Comment:
I'm good either way, I was iterating through some issues I was hitting, and cover-erase snuck in as I was trying to make sure that I wasn't reporting data from a previous run.